test/base.js

```diff
@@ -0,0 +1,139 @@
+/*
+ * LiskHQ/lisk-commander
+ * Copyright © 2017–2018 Lisk Foundation
+ *
+ * See the LICENSE file at the top-level directory of this distribution
+ * for licensing information.
+ *
+ * Unless otherwise agreed in a custom licensing agreement with the Lisk Foundation,
+ * no part of this software, including this file, may be copied, modified,
+ * propagated, or distributed except according to the terms contained in the
+ * LICENSE file.
+ *
+ * Removal or modification of this copyright notice is prohibited.
+ *
+ */
+import os from 'os';
+import BaseCommand, { defaultConfigFolder } from '../src/base';
+import * as config from '../src/utils/config';
+import * as printUtil from '../src/utils/print';
+
+describe('base command', () => {
+	let command;
+	beforeEach(() => {
+		command = new BaseCommand();
+		sandbox.stub(command, 'parse').returns({});
+		sandbox.stub(command, 'error');
+		sandbox.stub(config, 'getConfig').returns({});
+		return Promise.resolve();
+	});
+
+	describe('#init', () => {
+		it('should set XDG_CONFIG_HOME to default value', async () => {
+			delete process.env.LISK_COMMANDER_CONFIG_DIR;
```

Comment:
As with the previous PR regarding restoring environmental variables after the suite has run.